nav/config/config.go

```diff
@@ -174,3 +177,16 @@ func validateType(t *string) error {
 		return fmt.Errorf("invalid output type: %s\nChoose one of the following: graphOnly, jsonOutputPlain, jsonOutputB64 or jsonOutputGZB64", *t)
 	}
 }
+
+func validateGType(t *c.OutIMode) error {
+        switch *t {
+        case 0:
+                *t = c.DefaultGOutputType
+                fmt.Printf("No output format specified. Defaulting to %d.\n", c.DefaultGOutputType)
+                return nil
+        case c.OText, c.OPNG, c.OJPG, c.OSVG:
+                return nil
+        default:
+                return fmt.Errorf("invalid graphviz output type: %s\nChoose one of the following: 0, 1, 2, 3", *t)
```

Comment:
It would be nice to include what those integer options mean in the message, i.e. `...following: 0 (text), 1 (png)...`. Also, the suggested values (0-3) are inconsistent with the one defined below (1-4).

Reply:
This issue represents a regression that occurred when the configuration pull request was merged. The inconsistency can be attributed to historical reasons. In any case, it is something I failed to notice and must address. Your keen observation is much appreciated. Thank you for catching this.